tests/plugins/test_pin_operator_digests.py

```diff
@@ -1206,3 +1208,83 @@ def test_pullspecs_replacements(self, tmpdir, docker_tasker):
         }
 
         assert result['pin_operator_digest'] == expected
+
+    @responses.activate
+    def test_other_metadata_replacements(self, tmpdir, docker_tasker):
+        pull_specs = ['myimage:v1.2.2']
+        test_url = "https://example.com/modifications.json"
+        modification_data = {
+            "pullspec_replacements": PULLSPEC_REPLACEMENTS,
+            "append": {
+                "spec": {
+                    "relatedImages": [
```

Comment:
This is wrong example, OSBS is doing it, remove relatedImages from test

Reply:
ACK